code/game/objects/structures/lavaland/gulag_vent.dm

```diff
@@ -30,7 +30,14 @@
 	var/mob/living/living_user = user
 	occupied = TRUE
 	living_user.balloon_alert_to_viewers("hauling...")
-	var/succeeded = do_after(living_user, 8 SECONDS, src)
+
+	var/boulder_lift_speed = 8 SECONDS
+
+	var/obj/item/organ/internal/cyberimp/chest/spine/potential_spine = living_user.get_organ_slot(ORGAN_SLOT_SPINE)
```

Comment:
really all of these

Reply:
Would a helper proc work better? It seems like a lot of places to be sending signals from when it might be easier to just have a proc do most of the same stuff here but checking different values.

Reply:
Well, if you can find a way to reduce all these get_organ_slot() calls, it'd be appreciated.